core/src/main/scala/org/apache/spark/ui/UIUtils.scala

```diff
@@ -431,7 +431,7 @@ private[spark] object UIUtils extends Logging {
     }
 
     val headerRow: Seq[Node] = {
-      headers.view.zipWithIndex.map { x =>
+      headers.to(LazyList).zipWithIndex.map { x =>
```

Comment:
Just a question. Is this better?

Reply:
When using Scala 2.12, it is lazy, so I prefer to use LazyList to restore its lazy characteristics with Scala 2.13